onboard/catkin_ws/src/data_pub/scripts/zero_imu.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
```

Comment:
Not sure what this file does, we should check if it's something useful. Maybe this is the code that will reset the imu relative to where it's currently facing?

Reply:
Don't merge